### ISSUE TITLE
Color issue

### ISSUE THREAD

**Issue by NotACoin** [markusfisch/ScreenTime#7]
Hello,

I like the simplicity of this app, just what I needed ! But I encounter an issue I think.
The background of the wheel(clock) turned light blue as if I was on my phone the whole day. But it was turned off during the night and the clock indicate 7h in total which looks correct. If I move the starting day forward it go back to normal, but if I show the two days it turns light blue.
Is it a bug ? Is it Is it possible to check the data the app is using, and maybe change it by hand if it's wrong ?

Thanks

**Comment by markusfisch** [comment]
Well, that really sounds like a bug 🤔 Sorry for the inconvenience, and thanks for reporting!

Could you share a screenshot?

At the moment there's no way to check or edit the data, sorry.

**Comment by NotACoin** [comment]
Okay, this is two screenshot one for today and one for the two last days. Tomorrow I'll be able to check if it's only a day 2 issue or if it affect all days. 
My phone had no screen sleep timer (but I still put it on sleep manually), maybe it was the issue ? Now I changed that phone parameter and the screen turn off automatically after 15sec of inactivity.

![Screenshot_2022-04-14-09-51-06](https://user-images.githubusercontent.com/60425329/163339688-d525ad49-51e3-4fd8-b139-070df50e63d3.png)
![Screenshot_2022-04-14-09-51-12](https://user-images.githubusercontent.com/60425329/163339692-c5edcd7e-cb8b-4e5e-a303-019371d455d0.png)

**Comment by NotACoin** [comment]
Hello, just to update it looks like it's working well, and the problem was only on day 2 apparently. Imo setting the time limit of sleep mode to inifinite in the phone parameter was the source of the problem. How does the app calculate the screen time ?

**Comment by markusfisch** [comment]
Hm, interesting 🤔 Do you mean the sleep mode for the display?

The app is basically couting the time between [Intent.ACTION_USER_PRESENT](https://developer.android.com/reference/android/content/Intent#ACTION_USER_PRESENT) and [Intent.ACTION_SCREEN_OFF](https://developer.android.com/reference/android/content/Intent#ACTION_SCREEN_OFF) broadcasts.

I chose `Intent.ACTION_USER_PRESENT` over [Intent.ACTION_SCREEN_ON](https://developer.android.com/reference/android/content/Intent#ACTION_SCREEN_ON) because the latter is sent before the keyguard, and so before the device is in an interactive state.

**Comment by NotACoin** [comment]
Yes, I mean the sleep mode for the display. I'm not expert at all, maybe if the setting is on "never go on sleep mode automatically", the signal is never sent and when I push the sleep button (hardware) the signal is different ? Or bypass the ACTION_SCREEN_OFF ? I don't know if it makes sense.
